pkg/volume/api/api.go

```diff
@@ -0,0 +1,388 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/kubernetes-csi/csi-proxy/pkg/utils"
+	"k8s.io/klog/v2"
+)
+
+// API exposes the internal volume operations available in the server
```

Comment:
(Since we are refactoring quite a bit) Instead of calling this `API`, how about something a little more descriptive like `HostAPI` or `HostUtils` for the interface as well as package name?

Reply:
Makes sense. I like HostAPI. How about we do this in a separate PR for all API groups after the initial migration? I like batching similar refactors in one go.

Reply:
Tracking this in #242